sunpy/map/mapbase.py

```diff
@@ -1387,7 +1390,11 @@ def submap(self, bottom_left, top_right=None):
             object with units of `~astropy.units.pixel`.
         top_right : `astropy.units.Quantity` or `~astropy.coordinates.SkyCoord`
             The top_right coordinate of the rectangle. Can only be omitted if
-            ``bottom_left`` has shape ``(2,)``.
+            ``bottom_left`` has shape ``(2,)``, passing this as a positional argument is deprecated.
```

Comment:
I can't make this a suggestion, but edit the above lines.
```
        bottom_left : `astropy.units.Quantity` or `~astropy.coordinates.SkyCoord`
            The bottom-left coordinate of the rectangle. If a `SkyCoord` it can
            have shape ``(2,)`` and simultaneously define ``top_right``. If specifying
            pixel coordinates it must be given as an `~astropy.units.Quantity`
            object with units of `~astropy.units.pixel`.
        top_right : `astropy.units.Quantity` or `~astropy.coordinates.SkyCoord`
            The top-right coordinate of the rectangle.
            Passing this as a positional argument is deprecated.
```